modules/xml/xml-plugin.c

```diff
@@ -33,6 +33,11 @@ static Plugin xml_plugins[] =
     .name = "xml",
     .parser = &xml_parser,
   },
+  {
+    .type = LL_CONTEXT_PARSER,
+    .name = "windows-eventlog-parser",
+    .parser = &xml_parser,
```

Comment:
just noticed that we have an SCL with the same name.

scl/windowseventlog/plugin.conf

Reply:
Good catch! Renamed it to `windows-eventlog-xml-parser()`.